src/lib/prov/tpm/tpm.cpp

```diff
@@ -388,7 +388,6 @@ class TPM_Signing_Operation : public PK_Ops::Signature
          {
          if(auto* key = dynamic_cast<const TPM_PrivateKey*>(&spec.key()))
             {
-            const std::string padding = spec.padding();
```

Comment:
This shouldn't be removed, actually there is a missing check on the value - TPM can only sign with PKCS v1.5 padding so for example if padding was PSS this should throw as unsupported.